core/ErrorQueue.cc

```diff
@@ -6,88 +6,31 @@
 
 namespace sorbet::core {
 
-namespace {
-// In the case of location ties, determines which query response takes priority in the vector produced by
-// drainWithQueryResponses. Larger values means greater specificity.
-u2 getQueryResponseTypeSpecificity(const core::lsp::QueryResponse &q) {
-    if (q.isEdit()) {
-        // Only reported for autocomplete, and should take precedence over anything else reported
-        return 8;
-    } else if (q.isDefinition()) {
-        return 7;
-    } else if (q.isSend()) {
-        return 6;
-    } else if (q.isField()) {
-        return 5;
-    } else if (q.isIdent()) {
-        return 4;
-    } else if (q.isConstant()) {
-        return 3;
-    } else if (q.isLiteral()) {
-        return 2;
-    } else {
-        return 1;
-    }
-}
-} // namespace
-
 using namespace std;
 
 ErrorQueue::ErrorQueue(spdlog::logger &logger, spdlog::logger &tracer, shared_ptr<ErrorFlusher> errorFlusher)
     : errorFlusher(errorFlusher), owner(this_thread::get_id()), logger(logger), tracer(tracer){};
 
-pair<vector<unique_ptr<core::Error>>, vector<unique_ptr<core::lsp::QueryResponse>>>
-ErrorQueue::drainWithQueryResponses() {
+vector<unique_ptr<core::Error>> ErrorQueue::drainAllErrors() {
```

Comment:
Can we remove this function?

Reply:
It's used in a number of tests

Reply:
could remove it if `flushAllErrors` returned the vector instead